widget/embedded/src/types/config.ts

```diff
@@ -143,8 +140,30 @@ export type Features = Partial<
     | 'customTokens',
     'visible' | 'hidden'
   >
+>;
+
+/**
+ * `Routing`
+ *
+ * @property {'disabled' | 'enabled'} [avoidNativeFee]
+ * - The enablement state for the avoid native fee feature. Optional property.
+ *
+ * @property {number} [maxLength]
```

Comment:
I think we can express these comments more clearly
Or, for example, take it from the document here:

https://api-docs.rango.exchange/reference/getbestroutes

Reply:
fixed